protostar/starknet_gateway/gateway_facade.py

```diff
@@ -305,8 +305,7 @@ async def invoke(
         account_address: str,
         signer: BaseSigner,
         inputs: Optional[CairoOrPythonData] = None,
-        max_fee: Optional[int] = None,
-        auto_estimate_fee: bool = False,
+        max_fee: Fee = "auto",
```

Comment:
```suggestion
        max_fee: Fee,
```

Reply:
[make max_fee mandatory and update tests](https://github.com/software-mansion/protostar/pull/979/commits/b2e0a73bd1102a4cb0555cb2e3e91535b785d958)